packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2268,12 +2322,13 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
  
  ## Provide a k8s version to auto dashboard import script example: kubeTargetVersionOverride: 1.16.6
  ##
-@@ -76,8 +334,19 @@
+@@ -76,8 +334,20 @@
  
  ##
  global:
 +  cattle:
 +    systemDefaultRegistry: ""
++    clusterId: ""
```

Comment:
As you pointed out in the description, the external URLs we are using are `api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-grafana:80/proxy`, so there should be no need to add this value into the PR as it is right now.